CMakeLists.txt

```diff
@@ -1193,6 +1193,18 @@ ENDIF()
 
 set(NETDATA_COMMON_LIBRARIES ${NETDATA_COMMON_LIBRARIES} m ${CMAKE_THREAD_LIBS_INIT})
 
+# -----------------------------------------------------------------------------
+# ACLK
+
+file(STRINGS "${CMAKE_SOURCE_DIR}/config.h" DEFINE_ENABLE_ACLK REGEX "^#define ENABLE_ACLK 1$")
```

Comment:
I don't think this is correct way to fix this.
Since we are moving to `cmake` generating its own `config.h`
Also the regex provided wont match `#define ENABLE_ACLK` which would also enable it

Reply:
> Since we are moving to cmake generating its own config.h

This PR just fixes the compilation when `config.h` was generated by Autotools with `--disable-cloud`. I completely agree that CMake should generate `config.h` by itself. But it's not the purpose of this PR.